### ISSUE TITLE
Score tracker doesn't support two glyphs in same grid square

### ISSUE THREAD

**Issue by mglennon** [adwiii/nc-ftc-inspection-spark#21]
I was informed by my head referees that the following situation is a valid (two glyphs scored in the same grid square) but the software does not appear to have a way of entering this situation.
![20180224_091406](https://user-images.githubusercontent.com/5652065/36735018-2ce00752-1ba3-11e8-8d6b-fca0ae98facb.jpg)

**Comment by adwiii** [comment]
Yes, there are a few scoring configurations that cannot be accurately scored in real time, but _any_ legal scoring configuration can be handled in the review phase. This is because the Autonomous and Driver Controlled do not have a counter for the number of glyphs scored and instead have a picture of a cryptobox that allows exactly one glyph per space and then uses that picture to calculate the score. We were able to add a work around for a glyph taking up two spaces and filling a row (namely the chevrons on the right), but we have no simple fix to the situation in the picture.

However, it can be scored in the review phase, as the review phase has all of the same fields as the official scoresheet. The current solution is either 1) score it as one glyph (pick one, or the one that benefits the team) in the right column, and then ignore the other glyph until the review phase where you manually increase the counter although this means that throughout the match the audience will see a score slightly lower than the actual total, or 2) pretend like the brown glyph is in the center column until there is another glyph actually in the center column, although this will look weird to the audience because they see the brown glyph not where it is, yet the score will be accurate.

I would appreciate feedback on what the most sensible solution, both from a volunteer Score Tracker and an audience perspective. 

From our experience in NC, we found that most of the time a situation like this occurred during a live match the team would end up coming by and descoring one of the glyphs in the course of normal game play before the Score Tracker had time to decide how to handle it, but we do need to have a set way of doing so.
